src/lib/viewers/box3d/model3d/Model3D.scss

```diff
@@ -18,6 +18,7 @@ $arrow-top-bottom: 4px solid transparent;
 
     .bp-pullup {
         bottom: 100%;
+        width: fit-content;
```

Comment:
Does this only affect Chrome? IE 11 doesn't support this AFAIK

Reply:
I tested IE 11 and Edge, and it looks fine even before the change.

Reply:
I don't think we should use experimental CSS values. Does setting `white-space: nowrap;` on the parent work?